tests/storages_tests/rdb_tests/alembic_tests/create-db.py

```diff
@@ -0,0 +1,43 @@
+import sys
```

Comment:
How about renaming `create_db.py` ?

Reply:
Thanks! I renamed it in https://github.com/optuna/optuna/pull/2890/commits/34d8ef962981ce947bb2c45406ef170b8bf76183.